backend/src/main/kotlin/hu/bme/sch/cmsch/component/token/TokenEntity.kt

```diff
@@ -102,6 +102,20 @@ data class TokenEntity(
     @property:ImportFormat(ignore = false)
     var displayDescription: String = "",
 
+    @field:JsonView(value = [ Edit::class, Preview::class, FullDetails::class ])
+    @Column(nullable = true)
+    @property:GenerateInput(type = INPUT_TYPE_DATE, order = 11, label = "Scannelhető innentől")
```

Comment:
Add `default = "0"`

Reply:
I set the default value to null, that would mean that there is no start or end